integreat_cms/cms/templates/pages/page_form.html

```diff
@@ -102,7 +108,7 @@ <h1 class="heading">
                                         {{ other_language.translated_name }}
```

Comment:
```suggestion
                                        {{ other_language.translated_name }}
                                        <span class="fp fp-rounded fp-{{ other_language.primary_country_code }}"></span>
                                        {% if other_language.secondary_country_code %}
                                            <span class="fp fp-rounded fp-{{ other_language.secondary_country_code }}"></span>
                                        {% endif %}
```
Please add this change to the line 96 as well.